src/lang/de-DE.json

```
@@ -918,5 +918,8 @@
     "statusPageSpecialSlugDesc": "Spezieller Slug {0}: diese Seite wird angezeigt, wenn kein Slug angegeben wird",
     "ntfyPriorityHelptextAllEvents": "Alle Ereignisse werden mit der höchsten Priorität gesendet",
     "ntfyPriorityHelptextAllExceptDown": "Alle Ereignisse werden mit dieser Priorität gesendet, außer {0}-Ereignisse, die eine Priorität von {1} haben",
-    "What is a Remote Browser?": "Was ist ein Remote-Browser?"
+    "What is a Remote Browser?": "Was ist ein Remote-Browser?",
+    "callMeBotGet": "Bekomme den Endpoint für {0}.",
+    "callMeBotInfo": "Beachte, dass du evtl. ratenbegrenzt wirst. Beispiel: Du darfst nur einen Telegram call pro 65 sekunden versenden! Bitte benutze den Cleanup-Button, um die URL nutzen zu können.",
+    "callMeBotInvalidEndpoint": "The specifyed endpoint is invalid!"
```

Don't include translations here. This may cause hard to resolve merge conflicts with https://weblate.kuma.pet

```suggestion
    "What is a Remote Browser?": "Was ist ein Remote-Browser?"
```